admin/app.js

```diff
@@ -124,7 +125,8 @@ busboy.extend(app, {
 
 const allowedPath = (url) => (/^\/pupil-register\/pupil\/add-batch-pupils$/).test(url) ||
   (/^\/test-developer\/upload-new-form$/).test(url) ||
-  (/^\/service-manager\/upload-pupil-census\/upload$/).test(url)
+  (/^\/service-manager\/upload-pupil-census\/upload$/).test(url) ||
+  (/^\/check-form\/submit-check-form$/).test(url)
```

Comment:
What do you think of making the URL ''/check-form/upload'?  I dislike the generic 'submit', as it says little.

Reply:
@jon-shipley url changed to the suggested one.